README.md

```diff
@@ -28,6 +28,15 @@ Please raise a new pull request for new library additions and existing library u
  - Now CDNJS supports adding a library by a single package.json, you just need to add a valid CDNJS package.json with npm/git auto-update config, and remove its version field in package.json, then we'll handle the remaining works.
    - Notes that you should have a `filename` to point to the main file of a lib, if that file is not minified, please still use `filename.min.js` or `filename.min.css` structure naming, we'll do the minify job.
 
+## Adding a library by a single package.json
+
+* [Fork the cdnjs repo](https://github.com/cdnjs/cdnjs/fork) to your own GitHub account.
+* Click the "Branch: master" dropdown, enter the name of the library being added, and choose "Create branch <LIBRARY_NAME> from master".
+* Go to `ajax/libs` directory.
+* Click "Create new file".
+* Enter `<LIBRARY_NAME>/package.json`, compose a valid [`package.json`](https://github.com/cdnjs/cdnjs#create-or-update-packagejson) for the library being added, enter commit message and click "Commit new file".
+* Submit a pull request.
+
```

Comment:
@halt-hammerzeit Let's use "list" feature of Markdown so that they can be correctly rendered
Ref: https://guides.github.com/features/mastering-markdown/#syntax